translationCoreApps/ScripturePane
# Tests

#### This pull request addresses:

added coverage, linting, and stubbed out tests

#### How to test this pull request:

I moved files around so make sure the code still runs.
The tests will fail because we don't have any real tests yet.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/scripturepane/68)
